modules/Core/includes/endpoints/SyncMinecraftGroupsEndpoint.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+class SyncMinecraftGroupsEndpoint extends KeyAuthEndpoint {
+
+    public function __construct() {
+        $this->_route = 'minecraft/sync-groups';
+        $this->_module = 'Core';
+        $this->_description = 'Update a users groups based on added or removed groups from the Minecraft server';
+        $this->_method = 'POST';
+    }
+
+    public function execute(Nameless2API $api): void {
+        $api->validateParams($_POST, ['server_id', 'user']);
```

Comment:
```suggestion
    public function __construct() {
        $this->_route = 'minecraft/{user}/sync-groups';
        $this->_module = 'Core';
        $this->_description = 'Update a users groups based on added or removed groups from the Minecraft server';
        $this->_method = 'POST';
    }

    public function execute(Nameless2API $api, User $user): void {
        $api->validateParams($_POST, ['server_id']);
```

Cant also then remove line 22

Reply:
Good idea, I'll also apply this to the discord endpoint